### PR TITLE
[2135] Maintenance: skip CONFIRM_PRODUCTION

### PR DESCRIPTION
**Pull request by saliceti**
## Context
https://github.com/DFE-Digital/npq-registration/actions/runs/11738475177/job/32701068931

## Changes proposed in this pull request
Use the make ci target to skip the confirmation

## Guidance to review
Code review. It will be tested in production in ECF soon.

## Checklist

- [x] I have performed a self-review of my code, including formatting and typos
- [x] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [x] I have added the `Devops` label
- [x] I have attached the pull request to the trello card

### CODE REVIEW

**Comment by saliceti** [comment]
[Bug: CONFIRM_PRODUCTION in maintenance](https://trello.com/c/9CYM0MpB/2135-bug-confirmproduction-in-maintenance)

File: maintenance/action.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ runs:
     - name: Enable maintenance mode
       if: inputs.mode == 'enable'
       shell: bash
-      run: make ${{ inputs.environment }} maintenance-fail-over
+      run: make ci ${{ inputs.environment }} maintenance-fail-over
```
Comment (cwrw) [inline maintenance/action.yml:46]: can we add this to line 53 as well please?

**Review by neillturner — APPROVED** [review]
approved

File: maintenance/action.yml (hunk 1 above)
Comment (saliceti) [reply maintenance/action.yml:46]: done thanks

**Pull request merged**